tests/functional/tests_user_change_pass.py

```diff
@@ -0,0 +1,156 @@
+"""Testing ability to change password with existing users
+For getting valid response we need token of that user, old password and new password"""
+
+from tests.constants.constants import UserToTest, InvalidValues
+from tests.functional import ApiTestBase
+
+
+class TestChangePass(ApiTestBase):
+    """Testing server for ability to change pass with valid data and not valid data"""
+
+    def setUp(self):
+        """Get user token"""
+
+        super().setUp()
+        response = self.login(UserToTest.login, UserToTest.password)
+        self.token = response.json()['content']
+        self.assertEqual(200, response.status_code, "login error")
+
+    def test_change_pass_valid_data(self):
+        """Change pass with valid data"""
+
+        # change pass
+        valid_pass = UserToTest.password
+        change_pass = self.change_pass(self.token, UserToTest.password, valid_pass)
```

Comment:
You're actually changing password to the same blue as before, I would better choose some other value - in this case you would be able to verify that password was actually changed